azure/README.md

```diff
@@ -0,0 +1,197 @@
+
+```bash
+cd shared-state
+terraform init
+terraform apply
+```
+
+The storage account name should be in the output. Please use that to configure the Terraform backend in `main.tf` by uncommenting the part on the `backend "azurerm"`.
+
+**Note**: You'll need to follow the description below for the "existing resource group" i.e., use the `data.azurerm_resource_group.default` resource.
+
+
+### WrongSecrets-ctf-party
+
+1. Set either a new resource group or use an existing resource group in `main.tf` (it defaults to the existing `OWASP-Projects` resource group). Note that you'll need to find/replace references to "data.azurerm_resource_group.default" to "arurerm_resource_group.default" if you want to create a new one.
+2. check whether you have the right project by doing `az account show` (after `az login`). Want to set the project as your default? Use `az account set --subscription <.id here>`.
+3. If not yet enabled, register the required services for the subscription, run:
+    - `az provider register --namespace Microsoft.ContainerService`
+    - `az provider register --namespace Microsoft.KeyVault`
+    - `az provider register --namespace Microsoft.ManagedIdentity`
+4. Run `terraform init` (if required, use `tfenv` to select TF 0.14.0 or higher )
+5. Run `terraform plan` to see what will be created (optional).
+6. Run `terraform apply`. Note: the apply will take 5 to 20 minutes depending on the speed of the Azure backplane.
+7. Run `./build-and-deploy-azure.sh`. Your kubeconfig file will automatically be updated.
+
+Your AKS cluster should be visible in your resource group. Want a different region? You can modify `terraform.tfvars` or input it directly using the `region` variable in plan/apply.
+
+Are you done playing? Please run `terraform destroy` twice to clean up.
+
+### Test it
+
+When you have completed the installation steps, you can do `kubectl port-forward service/wrongsecrets-balancer 3000:3000` and then go to [http://localhost:3000](http://localhost:3000).
+
+Want to know how well your cluster is holding up? Check with
+
+```sh
+    kubectl top nodes
+    kubectl top pods
+```
+
+### Configuring CTFd
+
+You can use the [Juiceshop CTF CLI](https://github.com/juice-shop/juice-shop-ctf) to generate CTFd configuration files.
+
+Follow the following steps:
+
+```shell
+    npm install -g juice-shop-ctf-cli@9.1.2
+    juice-shop-ctf #choose ctfd and https://wrongsecrets-ctf.herokuapp.com as domain. No trailing slash! The key is 'test', by default feel free to enable hints. We do not support snippets or links/urls to code or hints.
+```
+
+Now visit the CTFd instance and setup your CTF. If you haven't set up a load balancer/ingress, the you can use `kubectl port-forward -n ctfd $(kubectl get pods --namespace ctfd -l "app.kubernetes.io/name=ctfd,app.kubernetes.io/instance=ctfd" -o jsonpath="{.items[0].metadata.name}") 8000:8000` and go to `localhost:8000` to visit CTFd.
```

Comment:
and how do i do this with the laodbalancer? e.g. what is the CTFD location in the curent setup?

Reply:
Edited this paragraph to make it less confusing. Testing locally should come first and loadbalancer is later. 
Sounds good now?